### DISCUSSION TITLE
Use the API Tokens to authenticate

### DISCUSSION THREAD

**Discussion by wolfwander** [dougiteixeira/proxmoxve#350] [Feature request]
Hi!

First of all, thanks a lot for this great Home Assistant Custom Integration that enables the monitoring of ProxmoxVE hosts, VMs and clusters.
Much better than the native Home Integration.

Could it be possible to use API Tokens to authenticate the Home Assistant ProxmoxVE Custom Integration?

**Comment by wolfwander** [comment]
Token authentication is already working for me.
Thansk a lot!

**Reply by valsergi** [reply]
Hello, since you did it, I can't get it to work. I updated the version and none of my 2 nodes now load. I have created the token but I can't get it to work. Can you help me? THANK YOU!
